public/js/chat.js

```diff
@@ -0,0 +1,34 @@
+const appendP = (output, comms, from, message) => {
```

Comment:
Remove 'comms' argument. [This](https://github.com/foundersandcoders/Live-Peers/blob/webrtc-boilerplate/public/js/comms.js) will be the implementation of Comms.js on the client side...

The chatroom/webrtc module constructors will take the **comms** instance, so for our comms.js module instead of this-
```
self.handlers[msg.app + '.' + msg.method](this, msg.from, msg.params);
```
The comms.js will be calling its handlers like this-
```
self.handlers[msg.app + '.' + msg.method](msg.from, msg.params);
```